lib/cisco_node_utils/bgp_af.rb

```diff
@@ -264,8 +258,7 @@ def default_advertise_l2vpn_evpn
 
     # dampen_igp_metric
     def dampen_igp_metric
-      result = config_get('bgp_af', 'dampen_igp_metric', @get_args)
-      result ? result.first.to_i : nil
+      config_get('bgp_af', 'dampen_igp_metric', @get_args)
```

Comment:
Hm, need to take another look at this one.

`auto_default` is used specifically for cases where we have a defined `default_value` but do NOT want the getter to automatically return the `default_value` in the case where the `config_get_token` fails to find a match (and would otherwise return `nil`). 

In this case, from the baseline code, the `default_value` is 600 seconds, but if the regexp fails to match, we want to return `nil` (dampen-igp-metric disabled, I guess?) rather than `600`. I'm not enough of an expert in this feature to understand why this asymmetry.